database/migrations/2023_09_14_213354_create_hotels_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('hotels', function (Blueprint $table) {
+            $table->integer('user_id');
```

Comment:
@anichukwuebuka65 Please review the tables that already exist and make any necessary corrections.

Reply:
I have made the changes. The migrations now run successfully.